joern-cli/src/main/scala/io/shiftleft/joern/JoernStats.scala

```diff
@@ -0,0 +1,42 @@
+package io.shiftleft.joern
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.{NodeTypes, nodes}
+import io.shiftleft.codepropertygraph.generated.nodes.MethodParameterIn
+import io.shiftleft.semanticcpg.language._
+import io.shiftleft.dataflowengineoss.language._
+import io.shiftleft.dataflowengineoss.queryengine.{EngineConfig, EngineContext}
+import io.shiftleft.dataflowengineoss.semanticsloader.Semantics
+import io.shiftleft.joern.console.JoernWorkspaceLoader
+import overflowdb.traversal.{Traversal, jIteratortoTraversal}
+
+case class StatsConfig(cpgFileName: String = "cpg.bin")
+
+object JoernStats extends App {
+  private def parseConfig: Option[StatsConfig] = {
+    new scopt.OptionParser[StatsConfig]("joern-stats") {
+      head("Prints statistics on various nodes found in Code Property Graphs")
+      help("help")
+
+      arg[String]("cpg")
```

Comment:
Let's make `cpg.bin` the default so that it is consistent with `joern-export` and `joern-flow`.